pkg/model/build_reason.go

```diff
@@ -53,6 +61,7 @@ var translations = map[BuildReason]string{
 	BuildReasonFlagTriggerCLI:     "CLI Trigger",
 	BuildReasonFlagTriggerUnknown: "Unknown Trigger",
 	BuildReasonFlagTiltfileArgs:   "Tilt Args",
+	BuildReasonFlagChangedDep:     "New Image",
```

Comment:
I'm not sure I as a user would understand what this meant... why not just "dependency updated" or something?

Reply:
i changed it to "image updated", since we don't really use the "dependency" language in other user-facing parts of tilt